src/islider_core.js

```diff
@@ -490,6 +407,145 @@ define('iSlider', [], function(){
     };
 
     /**
+    *  removeEventListener to release the memory
+    */
+    iSlider.prototype.release = function(){
```

Comment:
if iSlider have release method to remove the event listener,  it still need a method to destroy all the references and dom